### DISCUSSION TITLE
Nice_table: avoid rounding up and show the p-value as scientific notation?

### DISCUSSION THREAD

**Discussion by keithmeplz** [rempsyc/rempsyc#18] [Q&A] [answered]
Hi Remi, I have been using your remipsyc package in R to generate tables, to which I am extremely grateful. Thank you for having come up with such useful package. 

However, I have also been trying to modify the parameters in your package so that the p-value will not be rounded up and that it will appear as scientific notation. Could you please tell which function I should "fix"? I've gone through the source codes for nice_table and for format_p.

**Comment by rempsyc** [comment, marked as answer]
See solution in #19
